.eslintrc.js

```diff
@@ -53,5 +54,9 @@ module.exports = {
       },
     ],
   },
-  ignorePatterns: ["*.spec.ts", "webpack.config.js"],
+  ignorePatterns: [
+    "*.spec.ts",
+    "webpack.config.js",
+    "src/models/ViewConfigurationAdapter.ts",
```

Comment:
Why not lint this file?

Reply:
Because by the file's nature it's checking 'any' and seeing if it matches enum schemas, and that freaks out the linter. I have to do some research into how to type this puppy before i can let the linter at it.